tests/foreman/longrun/test_oscap.py

```diff
@@ -526,6 +551,145 @@ def test_positive_oscap_run_with_tailoring_file_and_capsule(self):
+        ssh.upload_file(
+            local_file=tailor_path,
+            remote_file="/tmp/{0}".format(file_name)
+        )
+        # Creates host_group for rhel7
+        make_hostgroup({
+            'content-source-id': self.proxy_id,
+            'name': hgrp7_name,
+            'organizations': self.config_env['org_name']
+        })
+
+        tailor_result = make_tailoringfile({
+            'name': tailoring_file_name,
+            'scap-file': '/tmp/{0}'.format(file_name),
+            'organization': self.config_env['org_name']
+        })
+        result = TailoringFiles.info({'name': tailoring_file_name})
+        self.assertEqual(result['name'], tailoring_file_name)
+        # Creates oscap_policy for rhel7.
+        scap_id, scap_profile_id = self.fetch_scap_and_profile_id(
+            policy_values.get('content'),
+            policy_values.get('profile')
+        )
+        Ansible.roles_import({'proxy-id': self.proxy_id})
+        Ansible.variables_import({'proxy-id': self.proxy_id})
+        role_id = Ansible.roles_list({'search': 'foreman_scap_client'})[0].get('id')
+        make_scap_policy({
+            'scap-content-id': scap_id,
+            'hostgroups': policy_values.get('hgrp'),
+            'deploy-by': 'ansible',
+            'name': policy_values.get('policy'),
+            'period': OSCAP_PERIOD['weekly'].lower(),
+            'scap-content-profile-id': scap_profile_id,
+            'weekday': OSCAP_WEEKDAY['friday'].lower(),
+            'tailoring-file-id': tailor_result['id'],
+            'tailoring-file-profile-id': tailor_result['tailoring-file-profiles'][0]['id'],
+            'organizations': self.config_env['org_name']
+        })
+        distro_os = vm_values.get('distro')
+        with VirtualMachine(distro=distro_os) as vm:
+            host_name, _, host_domain = vm.hostname.partition('.')
+            vm.install_katello_ca()
+            vm.register_contenthost(
+                self.config_env['org_name'],
+                self.config_env['ak_name'].get(distro_os)
+            )
+            self.assertTrue(vm.subscribed)
+            Host.set_parameter({
+                'host': vm.hostname.lower(),
+                'name': 'remote_execution_connect_by_ip',
+                'value': 'True',
+            })
+            vm.configure_rhel_repo(settings.rhel7_repo)
+            add_remote_execution_ssh_key(vm.ip_addr)
+            Host.update({
+                'name': vm.hostname.lower(),
+                'lifecycle-environment': self.config_env['env_name'],
+                'content-view': self.config_env['cv_name'],
+                'hostgroup': vm_values.get('hgrp'),
+                'openscap-proxy-id': self.proxy_id,
+                'organization': self.config_env['org_name'],
+                'ansible-role-ids': role_id
+            })
+            # needed to work around BZ#1656480
+            if bz_bug_is_open(1656480):
```

Comment:
Bugzilla is closed as duplicate of https://bugzilla.redhat.com/show_bug.cgi?id=1650103

Reply:
ahh forgot